docs/index.rst

```diff
@@ -10,10 +10,8 @@ Welcome to Chipyard's documentation!
 
 Chipyard is a framework for designing and evaluating full-system hardware using agile teams.
 It is composed of a collection of tools and libraries designed to provide an integration between open-source and commercial tools for the development of systems-on-chip.
-New to Chipyard? Jump to the :ref:`Chipyard Basics` page for more info.
 
-Quick Start
-===============================
+.. IMPORTANT:: **New to Chipyard?** Jump to the :ref:`Chipyard Basics` page for more info.
 
 System Requirements
```

Comment:
Don't we want to remove this stuff too?

Reply:
Yes we do but I wanted someone who knew if anything needed to be kept to make the deletion.
I'm not sure they are 100% redundant.

Reply:
They aren't completely 100% redundant right now. We should probably make the toolchains installation in initial-setup more similar to the current quick-start form (so peolpe who are just looking for codeline to copy will copy the correct one).
We should also copy the "what's next" part.